language/tools/cost-synthesis/src/stack_generator.rs

```diff
@@ -801,6 +806,10 @@ where
         if self.iters == 0 {
             return None;
         }
+        // Clear the VM's data cache -- otherwise we'll windup grabbing the data from
```

Comment:
@tzakian I moved this here as it was the easiest place given ownership.
I think that is semantically the same but please review